doc/architectural/howto.md

```diff
@@ -16,12 +16,27 @@ to the basic data structures and workflow needed for contributing to
 
 ## Initial setup
 
-Clone the [CBMC repository][cbmc-repo] and build it:
+Clone the [CBMC repository][cbmc-repo] and build it. The build instructions are
+written in a file called COMPILING.md in the top level of the repository. I
+recommend that you build using CMake---this will place all of the CBMC tools in
```

Comment:
Still mention that `make` exists for those who need it

Reply:
Done, thank you